plugins/draco/CMakeLists.txt

```diff
@@ -25,6 +25,19 @@ f3d_plugin_declare_reader(
   FORMAT_DESCRIPTION "Draco"
 )
 
+# Needs https://gitlab.kitware.com/vtk/vtk/-/merge_requests/10884
+if(VTK_VERSION VERSION_GREATER_EQUAL 9.3.20240214)
+  f3d_plugin_declare_reader(
+    NAME GLTFDraco
+    SCORE 90
```

Comment:
BTW there is no score on the draco reader above.

Reply:
It relies on the default score, which is fine?

Reply:
ok